### ISSUE TITLE
Table group row cannot be fixed using TableFixedColumns

### ISSUE THREAD

**Issue by ushkal** [DevExpress/devextreme-reactive#1798]
- [x] I have searched this repository's [issues](https://github.com/devexpress/devextreme-reactive/issues) and believe that this is not a duplicate.

### I'm using ...

- [x] React Grid
- [ ] React Chart
- [ ] React Scheduler

### Current Behaviour
The group row content is not fixed when the TableGroupRow.COLUMN_TYPE is specified as a fixed column type.

### Expected Behaviour
The group row content should be fixed

### Steps to Reproduce (for Bugs)

https://codesandbox.io/s/r4kvv4605p

### Environment

- devextreme-reactive: 1.10.0
- react: 16.7.0
- browser: xxx
- bootstrap: none
- react-bootstrap: none
- material-ui: none

**Comment by attiqeurrehman** [comment]
@ushkal any update on this.

**Comment by patrick-moore** [comment]
Not ideal, but this custom GroupCell component is working for me right now. Posting here in case it helps anyone:

in TableGroupRow:

```tsx
<TableGroupRow cellComponent={GroupCell} />
```

Custom GroupCell component:

```tsx
import * as React from "react";
import { TableGroupRow } from "@devexpress/dx-react-grid-bootstrap3";
import styled from "styled-components";
import { Icon } from "../../core/icons";

export const GroupCell: React.SFC<TableGroupRow.CellProps> = props => {
  return (
    <GroupCellContainer colSpan={props.colSpan} onClick={props.onToggle}>
      <GroupCellContent
        level={props.tableRow.row.compoundKey.split("|").length - 1}
      >
        <GroupCellIcon name={props.expanded ? "caret-down" : "caret-right"} />
        {props.column.title}

        <strong>
          {": "}
          {props.row.value}
        </strong>
      </GroupCellContent>
    </GroupCellContainer>
  );
};

const GroupCellContainer = styled.td`
  cursor: pointer;
`;

const GroupCellIcon = styled(Icon)`
  width: 16px;
  margin: 4px;
`;

const GroupCellContent = styled.div<{ level: number }>`
  position: sticky;
  left: ${props => 12 + props.level * 33}px;
  display: inline-flex;
  align-items: baseline;
  justify-content: space-around;
`;

```

**Comment by ushkal** [comment]
Hi ,

We have implemented this feature in v1.11.1. Now the group row content always sticks to the left edge.

![group-row-fix](https://user-images.githubusercontent.com/44223690/59512179-ddf1ff00-8ec0-11e9-9fdf-d409cd22220d.gif)
